x-pack/plugins/security_solution/cypress/e2e/detection_alerts/enrichments.cy.ts

```diff
@@ -29,7 +29,7 @@ import { login, visit } from '../../tasks/login';
 
 import { ALERTS_URL } from '../../urls/navigation';
 
-describe('Enrichment', () => {
+describe.skip('Enrichment', () => {
```

Comment:
this test has been failing for the last 3 builds and also fails locally

Reply:
Could you please create a ticket similar to this one https://github.com/elastic/kibana/issues/162818 and put there details about skipped test and link to the failing builkite job? Thanks!

Reply:
[ticket](https://github.com/elastic/kibana/issues/163567) created!

Reply:
@PhilippeOberti I was thinking more about this and was wondering if this test is failing only on your PR? If that is the case and the test is not failing on main then I would say we should figure out why these changes cause this test to fail and fix it instead of skipping.

Reply:
thanks for coming back to this. I spent a bit more time and was able to actually fix the test. It was NOT flaky. I guess I was confused the first time.
The test is now unskipped and fixed. I also closed the ticket I opened. Sorry!

Reply:
Thank you so much!